components/datetime/src/options/components.rs

```diff
@@ -368,10 +382,23 @@ pub enum Month {
 // Each enum variant is documented with the UTS 35 field information from:
 // https://unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table
 //
-/// Options for displaying a time zone for the `components::`[`Bag`].
-///
-/// Note that the initial implementation is focusing on only supporting ECMA-402 compatible
-/// options.
+/// Options for displaying the current week for the `components::`[`Bag`].
+#[doc(hidden)]
+// TODO(#488): make visible once fully supported.
+#[derive(Debug, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum Week {
+    /// The week of the month, such as "3".
+    #[cfg_attr(feature = "serde", serde(rename = "week-of-month"))]
```

Comment:
`question`: can you use `rename_all="kebab-case"`?

Reply:
Yes that works. Done.